src/vishakha/BankOperation.java

```diff
@@ -0,0 +1,153 @@
+package vishakha;
+
+import java.util.Scanner;
+
+public class BankOperation {
+
+	int iciciBal = 8000;
+	int hdfcBal = 5000;
+	int bobBal = 10000;
+	int sbiBal = 1000;
+	int iciciCCount;
+	int hdfcCCount;
+	int bobCCount;
+	int sbiCCount;
+	int iciciDCount;
+	int hdfcDCount;
+	int bobDCount;
+	int sbiDCount;
+	int totalBalance;
+	static int creditTotalCount;
+	static int debitTotalCount;
+	static String bankName;
+	static String operation;
+	static int amount;
+	
+	void bankOperation(String bankName, String op, int amount){
+		switch(op){
+		case "credit":
```

Comment:
instead of writing like this you can use op.toUpperCase(), not sure we learned or not.